integration/test/test_ompt.py

```diff
@@ -53,7 +53,7 @@ def setUpClass(cls):
                          '.' + cls.__name__ + ') ...')
         test_name = 'test_ompt'
         cls._report_path = '{}.report'.format(test_name)
-        cls._agent_conf_path = 'test_' + test_name + '-agent-config.json'
+        cls._agent_conf_path = test_name + '-agent-config.json'
```

Comment:
Was this change to the OMPT code intentionally included?

Reply:
It is a copy paste error that I found when copy pasting again. This issue may be in other files too...I can check.